types/src/state_store/state_value.rs

```diff
@@ -31,24 +30,24 @@ use serde::{Deserialize, Deserializer, Serialize, Serializer};
 )]
 pub enum StateValueMetadata {
     V0 {
-        payer: AccountAddress,
         deposit: u64,
         creation_time_usecs: u64,
     },
 }
 
 impl StateValueMetadata {
-    pub fn new(
-        payer: AccountAddress,
-        deposit: u64,
-        creation_time_usecs: &CurrentTimeMicroseconds,
-    ) -> Self {
+    pub fn new(deposit: u64, creation_time_usecs: &CurrentTimeMicroseconds) -> Self {
```

Comment:
so this thing is never used?

Reply:
Right, only on devnet, never on testnet or mainnet.